clusterloader2/pkg/test/interface.go

```diff
@@ -25,6 +25,18 @@ import (
 // CreatContextFunc a type for function that creates Context based on given framework client and state.
 type CreatContextFunc func(f *framework.Framework, s *state.NamespacesState) Context
 
+// OperationType is a type of operation to be performed on an object.
+type OperationType int
+
+const (
+	// CREATE_OBJECT is create object operation.
+	CREATE_OBJECT = Operation(0)
```

Comment:
Shouldn't this be OperationType from above?

Reply:
Should be. Last moment rename and it was not propagated properly :/